tests/test_Box.py

```diff
@@ -96,6 +101,17 @@ def test_relationships(self):
         r = b4.relate(b1)
         self.assertEqual(r, DISJOINT)
 
+    def test_vectorized_contains(self):
+        b = Box.fromDegrees(200, 10, 300, 20)
+        x = np.random.rand(5, 3)
+        y = np.random.rand(5, 3)
+        z = np.random.rand(5, 3)
+        c = b.contains(x, y, z)
```

Comment:
Should also test non-contiguous memory here (and similar below).  Something like `b.contains(x[::2], y[::2], z[::2])`.